### ISSUE TITLE
"WebRTC Server not available" error after migrating HAS

### ISSUE THREAD

**Issue by rusty-genius** [AlexxIT/WebRTC#509]
Hello,

I migrated HAS from an RPI4 to a generic PC and everything works fine.

However, the custom WebRTC card always gives me an error when trying to show an rtsp stream.

This happens regardless from the underlying camera. It worked before the migration though.

I reinstalled the webrtc integration and even hacs itself but nothing seems to work.

When I look at the logs, i usually see this mistake (I have no idea if it belongs to this specific error)

I am no expert so I hope someone can help in this matter. Thanks!
```
Logger: frontend.js.latest.202306080
Source: components/system_log/__init__.py:257
First occurred: 11:10:23 (94 occurrences)
Last logged: 12:15:02

blob:http://homeassistant.local:8123/1fbd3798-1522-4650-9868-b6f6f3ed2285:1:100 ReferenceError: t is not defined
blob:http://homeassistant.local:8123/2d698d0e-bd91-4bf7-a596-f5216c79d03e:1:100 ReferenceError: t is not defined
blob:http://homeassistant.local:8123/d845f96b-0cf0-448e-95b6-3571570de5ec:1:100 ReferenceError: t is not defined
blob:http://homeassistant.local:8123/8bf97cd0-8811-4e8c-a787-0797eba44624:1:100 ReferenceError: t is not defined
blob:http://homeassistant.local:8123/19e0a8d4-0d8f-4e1c-a323-007aba3e2ad7:1:100 ReferenceError: t is not defined
```
![1](https://github.com/AlexxIT/WebRTC/assets/37304346/3247bf01-11eb-44cc-811c-7719cfa856d9)

**Comment by rusty-genius** [comment]
No one able to help with this? There must be someone else out there with this problem I guess?

**Comment by intgyl** [comment]
> No one able to help with this? There must be someone else out there with this problem I guess?

I have the same issue
Modify the go2rtc version to 1.2.0 and reboot HA , it works ok.

**Comment by rusty-genius** [comment]
> > No one able to help with this? There must be someone else out there with this problem I guess?
> 
> I have the same issue Modify the go2rtc version to 1.2.0 and reboot HA , it works ok.

Hey, thanks for your answer.

Could you give me a hint as to how I modify the go2rtc version?

In my Studio Code Server I have a got2rtc.yaml but this file only includes 2 lines with api and rtsp and the go2rtc-1.5.0 file is not editable. 

Can you help me out? thanks!

**Comment by intgyl** [comment]
> > > No one able to help with this? There must be someone else out there with this problem I guess?
> > 
> > 
> > I have the same issue Modify the go2rtc version to 1.2.0 and reboot HA , it works ok.
> 
> Hey, thanks for your answer.
> 
> Could you give me a hint as to how I modify the go2rtc version?
> 
> In my Studio Code Server I have a got2rtc.yaml but this file only includes 2 lines with api and rtsp and the go2rtc-1.5.0 file is not editable.
> 
> Can you help me out? thanks!

<img width="1231" alt="image" src="https://github.com/AlexxIT/WebRTC/assets/8155573/f89c5d80-035d-490d-8f23-30e1e7db7047">

In file utils.py,  BINARY_VERSION

**Comment by AlexxIT** [comment]
You can increase log level to check fail reason.

**Comment by gideondon** [comment]
I've had exaclty the same issue. followed these steps and it works again.
Took me so much time trying to find out.

**Comment by flyman1664** [comment]
Same issue on my side after migrating HAOS from Rpi3 to N100 mini pc

**Comment by flyman1664** [comment]
After uninstalling and reinstalling webrtc it is still not working.

![image](https://github.com/AlexxIT/WebRTC/assets/49916159/06015988-81eb-417c-a1ae-8f84c1811a6b)

**Comment by flyman1664** [comment]
Hi,

Since v3.4.0 update, all seems to be back to normal.
Thanks

**Comment by mylbeee** [comment]
Had this same issue. Noticed, that nothing was running at :1984. Resolved by manually deleting "go2rtc-1.8.1" file from the Home Assistant config-directory, and then re-adding the WebRTC device via GUI.

**Comment by TLongstride** [comment]
> Had this same issue. Noticed, that nothing was running at :1984. Resolved by manually deleting "go2rtc-1.8.1" file from the Home Assistant config-directory, and then re-adding the WebRTC device via GUI.

Thinks it the best solution, it work for me

**Comment by ness7288** [comment]
> config-directory, and then re-adding the 

also work for me, great thanks!!!

**Comment by nsimons** [comment]
> Had this same issue. Noticed, that nothing was running at :1984. Resolved by manually deleting "go2rtc-1.8.1" file from the Home Assistant config-directory, and then re-adding the WebRTC device via GUI.

Can also confirm this worked for me.

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/WebRTC/releases/tag/v3.6.0
